app/views/courses/index.html.erb

```diff
@@ -1,4 +1,4 @@
-<table id="courses" class="datatable display table table-striped table-bordered" >
+<table id="courses" class="datatable table table-striped table-bordered" >
```

Comment:
IIRC, some tables were styled as generic_datatable to begin with, and were initialized by class (rather than by id) in `app.js.coffee`.  I moved that to `datatables_defaults.js` as part of this update.  Other datatables are initialized separately, as in this case (in `courses.js.coffee`), if they are customized instead of relying only on the defaults.